src/stactools/nrcan_landcover/stac.py

```diff
@@ -89,14 +134,37 @@ def create_item(metadata: Dict[str, Any],
             title="Land cover of Canada metadata",
         ),
     )
+    item.add_asset(
```

Comment:
Since `thumbnail_url` is optional, should this be in an `if thumbnail_url:`?

Reply:
It defaults to `THUMBNAIL_HREF` if not specified